src/data/docs/documentation.md

```diff
@@ -5,257 +5,232 @@ description: How to write Unison documentation and associate it with your code
 
 # Documenting Unison Code
 
-Unison documentation is written in Unison. Documentation consists of values of the type `builtin.Doc`, and there's a special syntax to create these values:
+Unison ships with a powerful documentation format. Capable of expressing far more than markdown or static text, the documentation format empowers developers to write rich, clear, and correct docs. Documentation is a first class feature of the Unison language and is captured by the Unison type: `builtin.Doc2`. Unison docs offer native support for features like executing typechecked code samples, embedding docs within other docs, and annotating your writing with tooltips, asides, and callouts. We'll walk through some of these features next. 
```

Comment:
In the final release, the type will be `Doc`, and not `Doc2`, I think. Easy to find/replace, but let's not forget to change it.